cluster/mesos/docker/mesos-slave/bin/wrapdocker

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+# Copyright 2013-2014 Jérôme Petazzoni, Johan Haleby, lalyos, James Harris,
```

Comment:
I think we can just pull this out into our own repo and publish the mesos-slave dind image. That way k8s just has to download it.